patterns/index.md

```diff
@@ -0,0 +1,255 @@
+
+---
+layout: spec
+title: Frictionless Data Patterns
+listed: true
+updated: 11 December 2017
+created: 11 December 2017
+ietf-keywords: true
+author:
+ - Paul Walsh (Open Knowledge International)
+summary: A collection of patterns for frictionless handling of data.
+ietf-keywords: true
+---
+
+[issues]: https://github.com/frictionlessdata/specs/issues
+[repo]: https://github.com/frictionlessdata/specs
+
+### Table of Contents
+{:.no_toc}
+
+* Will be replaced with the ToC, excluding the "Contents" header
+{:toc}
+
+# Private properties
+
+## Overview
+
+Some software that implements the Frictionless Data specifications may need to store additional information on the various Frictionless Data descriptors.
+
+For example, a data registry that provides metatdata via `datapackage.json` may wish to set an internal version or identifier that is system-specific, and should not be considered as part of the user-generated metadata.
+
+Properties to store such information should be considered "private", and by convention, the names should be prefixed by an underscore `_`.
+
+## Implementations
+
+There are no known implementations at present.
+
+## Specification
+
+On any Frictionless Data descriptor, data that is not generated by the author/contributors, but is generated by software/a system handling the data, `SHOULD` be considered as "private", and be prefixed by an underscore `_`.
```

Comment:
"data" here is a bit confusing. I initially thought it talks about the csv or other data placed in resources.